packages/syft/src/syft/registry.py

```diff
@@ -54,13 +54,25 @@ def check_network(network: Dict) -> Optional[Dict[Any, Any]]:
             # networks without frontend have a /ping route in 0.7.0
             if not online:
                 try:
-                    url += "ping"
-                    res = requests.get(url, timeout=0.5)
+                    ping_url = url + "ping"
+                    res = requests.get(ping_url, timeout=0.5)
                     online = res.status_code == 200
                 except Exception:
                     online = False
 
             if online:
+                # Check if syft version was described in NetworkRegistry
+                if not network.get("version", None):
```

Comment:
@IonesioJunior should we have a check for `unknown` also here ? I'm thinking of the situation when a network node is online but unhealthy and therefore version is set to unknown, but when the node becomes healthy we should update the version. Thoughts?

Reply:
Oh, good catch! Thank you for noticing it! ❤️

Reply:
Is that what you meant? [ecc0cf6](https://github.com/OpenMined/PySyft/pull/6866/commits/ecc0cf6d71fe6e023641586cb9cc628d01495473)

Reply:
Yes, this looks good 🙌🏾